hack/jenkins/e2e.sh

```diff
@@ -47,6 +47,17 @@ function join_regex_no_empty() {
     fi
 }
 
+# GCP Project to fetch Trusty images.
+TRUSTY_IMAGE_PROJECT="${TRUSTY_IMAGE_PROJECT:-""}"
+
+# Get the latest Trusty image for a Jenkins job.
+function get_latest_trusty_image() {
+    echo "$(gsutil cat gs://${TRUSTY_IMAGE_PROJECT}/image-indices/latest-test-image-${JOB_NAME})"
+    # Clean up gsutil artifacts otherwise the later test stage will complain.
+    rm -rf .config > /dev/null 2>&1
```

Comment:
doesn't need to be fixed, but fyi you can write this even shorter as
`rm -rf .config &>/dev/null`